packages/mediacenter/kodi/patches/kodi-100.09-use-a-wrapper-to-setup-service-addons.patch

```diff
@@ -37,7 +37,7 @@ index f6aabfc615..496e1fcfa0 100644
 +      contextStr = "pre-uninstall";
 +      break;
 +    default:
-+      contextStr = StringUtils::Format("unknown(%d)", context);
++      contextStr = StringUtils::Format("unknown(%d)", static_cast<ADDON::LE_ADDON_CONTEXT>(context));
```

Comment:
Are you sure this is even needed with my patch?

Reply:
> Are you sure this is even needed with my patch?

Changed %d to {} as well.

Reply:
You are essentially doing nothing here though, you are casting to your own type. I wouldn't expect this to have any effect. If anything I would expect you to have to use `static_cast<int>` instead.

Reply:
That’s what I had thought too. I tried that first (and it compiled) then looking at https://github.com/xbmc/xbmc/pull/23453/commits/c82006b575b78efbb3f5aff40a159b90f245ea9d - followed that example. It also compiled.

is the `const` the issue that upsets stringutils::Format?

Reply:
Yea it could be the const as the format_as doesn't have a method for const enum.